src/utils/index.js

```diff
@@ -1,21 +1,45 @@
 const MAX_VALUE_LENGTH = 5;
 
 /**
- * 콘솔 입력에 대한 유효성 검사 함수
+ * 자동차 이름들에 대한 유효성 검사 함수
  *
  * @param {string | undefined} input 입력 받은 값
  * @returns 해당 입력값이 유효한지 여부
  */
-export function isInputValid(input) {
-  const hasSplitedValues = !!input && input.split(',').length > 0;
-  const hasNonEmptyValues = input
-    .split(',')
-    .every(
-      (splitedInput) =>
-        !!splitedInput &&
-        splitedInput.trim().length > 0 &&
-        splitedInput.trim().length <= MAX_VALUE_LENGTH,
-    );
+export function isCarNamesValid(input) {
+  if (!input) return false;
 
-  return hasSplitedValues && hasNonEmptyValues;
+  const splitedValues = input.split(',');
```

Comment:
밑에 로직을 보면 `trim()`이 반복해서 들어가고 있는데 여기에서 `trim()`을 미리 해줘도 좋을거 같네요
```suggestion
  const splitedValues = input.split(',').map((it) => it.trim());
```